docs/topics/wildfly-swarm/docs/reference/fractions/logging.adoc

```diff
@@ -30,6 +30,9 @@ A filter expression value to define a filter. Example for a filter that does not
 swarm.logging.async-handlers._KEY_.level:: 
 The log level specifying which message levels will be logged by this handler. Message levels lower than this value will be discarded.
 
+swarm.logging.async-handlers._KEY_.name:: 
+The name of the handler.
+
 swarm.logging.async-handlers._KEY_.overflow-action:: 
 Specify what action to take when the overflowing.  The valid options are 'block' and 'discard'
```

Comment:
singled quoted values should use backticks

Reply:
Note to self: not easily fixable in our sources.